sky/backends/backend_utils.py

```diff
@@ -2015,9 +2015,10 @@ class CloudFilter(enum.Enum):
 
 
 def get_clusters(
-        include_reserved: bool,
-        refresh: bool,
-        cloud_filter: str = CloudFilter.CLOUDS_AND_DOCKER
+    include_reserved: bool,
+    refresh: bool,
+    cloud_filter: str = CloudFilter.CLOUDS_AND_DOCKER,
+    cluster_names: Optional[Union[str, List[str]]] = None,
 ) -> List[Dict[str, Any]]:
     """Returns a list of cached cluster records.
```

Comment:
```suggestion
    """Returns a list of cached or optionally refreshed cluster records.
```

L2025-2026 need to be revised too.

Reply:
Done. Thanks!